pkg/api/secretgenerator/secretgenerator_test.go

```diff
@@ -0,0 +1,225 @@
+package secretgenerator
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sort"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"sigs.k8s.io/yaml"
+)
+
+func TestLoadConfigFromPath(t *testing.T) {
+	testcases := []struct {
+		name     string
+		config   string
+		expected string
+	}{
+		{
+			name: "No parameters",
+			config: `- item_name: Item1
```

Comment:
What I meant was not to inline it but to read it from a literal file and write it out, i.E. using pkg/testhelper.CompareWithFixture